css/collections-page.styl

```diff
@@ -70,40 +70,56 @@ $collection-viewer-button
     @media (max-width: 900px)
       margin: 0.6em 0
 
-  .collections-show
-    .collection-buttons-container
-      display: flex
-      justify-content: flex-end
-      margin-bottom: 2em
+  .collection__description-container
```

Comment:
Can we add a bottom margin to this, maybe 1em? Depending on the subjects and description length, the description can run right up against the subject:

<img width="689" alt="screen shot 2017-11-29 at 10 11 10 am" src="https://user-images.githubusercontent.com/5016731/33385249-c1c75980-d4ed-11e7-88b4-b480fe7b3a20.png">